docs/features/techdocs/configuration.md

```diff
@@ -158,6 +158,12 @@ techdocs:
       # (Required) Azure Blob Storage Container Name
       containerName: 'techdocs-storage'
 
+      # (Optional) Azure blob storage connection string.
+      # Can be useful for local testing through azurite
+      # Defaults to undefined
+      # if provided, takes higher priority, 'techdocs.publisher.azureBlobStorage.credentials' will become irrelevant
+      connectionString: ''
```

Comment:
When you add config entries, they need to also be added to the configuration schema, [in this file](https://github.com/backstage/backstage/blob/892a4b6c715b7fea5e0b24356d2cdb7e7a4343e8/plugins/techdocs-backend/config.d.ts#L199). You should mark it as `@visibility secret` just like `credentials` is, so that it doesn't accidentally leak sensitive details